### DISCUSSION TITLE
STM32 arm-none-eabi-gcc.exe issue

### DISCUSSION THREAD

**Discussion by amtrudea** [micropython/micropython#13675] [STM32 / Pyboard]
Hello all,

  I have been running into the following issue. I installed arm-none-eabi-gcc as well and it shows up in my PATH variables. I am unsure on how to fix this issue, any guidance would be greatly appreciated!
  
LINK build-NUCLEO_F091RC/firmware.elf
arm-none-eabi-gcc.exe: error: buil: No such file or directory
make: *** [Makefile:579: build-NUCLEO_F091RC/firmware.elf] Error 1

**Comment by ned-pcs** [comment]
I was able to do a build successfully. Do this to collect the whole log and post it so someone can help:
```sh
make BOARD=NUCLEO_F091RC clean submodules
make BOARD=NUCLEO_F091RC V=2 > make_output.txt
```

**Reply by rkompass** [reply]
>  buil: No such file or directory

could it be a typo in the makefile where there should be **build** for the build directory?

**Comment by jimmo** [comment]
@amtrudea Guessing you're compiling with Windows -- can you provide more context about how you're running the build, which instructions you're following, etc?
